### PR TITLE
Convert readme to md

### LINKED ISSUE #204: Use 'all contributors'
as in here https://github.com/s-weigand/flake8-nb

### PR DESCRIPTION
**Pull request by s-weigand**
This is the preparation needed for #204 (details see https://github.com/nbQA-dev/nbQA/pull/205#issuecomment-690159265).
I also requested that @MarcoGorelli [installs the all-contributor bot](https://github.com/apps/allcontributors/installations/new), since it is supposed (didn't test it yet) to [support a free language form](https://allcontributors.org/docs/en/bot/usage).
The [github action also uses some NLP](https://github.com/sinchang/all-contributors-action/blob/master/lib/parseComment.js), but since the bot is longer around with more people involved my guess is that it is more refined.

There is a minor [style change in the readme on github](https://github.com/s-weigand/nbQA/tree/convert-readme-to-md), mainly since github ignored the `400px` width in the `*.rst`, when rendering it.
Atm the all-contributors badge on my fork branch is red since the bot will add the `.all-contributorsrc` file and thus it is missing on my forges branch.

closes #204

### CODE REVIEW

**Comment by s-weigand** [comment]
Btw. just out of curiosity, why is `sphinx` pinned to version `3.1.2` was there a problem? 😄 
https://github.com/nbQA-dev/nbQA/blob/563870e25d87527d5f3c9d180b70f4ebe0fc54ba/docs/requirements-docs.txt#L1

File: setup.py [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
 
 from setuptools import find_packages, setup
 
-with open("README.rst", encoding="utf8") as readme_file:
-    readme = readme_file.read()
-    readme = readme.replace(
-        '.. raw:: html\n\n    <p align="center">\n        <a href="#readme">\n            <img alt="demo" src="https://raw.githubusercontent.com/nbQA-dev/nbQA-demo/master/demo.gif">\n        </a>\n    </p>\n\n',  # noqa
-        "",
-    )
+readme = Path("README.md").read_text(encoding="utf8")
```
Comment (MarcoGorelli) [inline setup.py:8]: :joy: this is wonderful

Comment (s-weigand) [reply setup.py:8]: 😉

**Review by MarcoGorelli — APPROVED** [review]
Awesome, thank you @s-weigand !

No reason to pin sphinx, we can unpin it :)

I've installed the bot - am I OK to merge this now or is there something else I need to do first?

**Comment by s-weigand** [comment]
I unpinned Sphinx and you are good to merger if you want ^^
After merge we can talk to the bot.

**Comment by s-weigand** [comment]
@all-contributors please add @MarcoGorelli  for code maintenance review test ideas

**Comment by s-weigand** [comment]
[List of builtin contribution types](https://allcontributors.org/docs/en/emoji-key).

**Pull request merged**